src/main/java/salesforce/utlis/strategy/IFeaturesPage.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
```

Comment:
The name of package should be utils and not utlis

Reply:
oksss